internal/biz/common.go

```diff
@@ -49,13 +50,13 @@ func DefaultRelationshipSendEvent(ctx context.Context, m *model.Relationship, ev
 	return nil
 }
 
-func DefaultSetWorkspace(ctx context.Context, namespace string, model *model.Resource, authz authzapi.Authorizer) error {
-	_, err := authz.SetWorkspace(ctx, model.Reporter.LocalResourceId, model.WorkspaceId, namespace, model.ResourceType)
+func DefaultSetWorkspace(ctx context.Context, namespace string, model *model.Resource, authz authzapi.Authorizer) (string, error) {
+	r, err := authz.SetWorkspace(ctx, model.Reporter.LocalResourceId, model.WorkspaceId, namespace, model.ResourceType)
 	if err != nil {
-		return err
+		return "", err
 	}
 
-	return nil
+	return r.ConsistencyToken.Token, nil
```

Comment:
Can `r.ConsistencyToken` can be null? Do we need to add a check?

Reply:
Right now, probably not, but it's impl-specific. Went ahead and added the check. :+1: